src/main/scala/ch/openolitor/stammdaten/StammdatenCommandHandler.scala

```diff
@@ -791,28 +769,70 @@ trait StammdatenCommandHandler extends CommandHandler with StammdatenDBMappings
 
   def updateKunde(idFactory: IdFactory, meta: EventTransactionMetadata, kundeId: KundeId, kunde: KundeModify) = {
     DB readOnly { implicit session =>
-      val personen = kunde.ansprechpersonen.map { person =>
+      if (isEmailUnique(idFactory, meta, Some(kundeId), kunde)) {
+        updateKundeEntity(idFactory, meta.originator, kundeId, kunde)
+      } else {
+        Failure(new InvalidStateException(s"Die übermittelte E-Mail Adresse wird bereits von einer anderen Person verwendet."))
```

Comment:
We should use the same language for all error messages in case we will translate them on the UI (sometime in the future). This message was already used and the one below (in English) is the same....

Reply:
I would suggest english for accessibility but most of them are in german. what do you think?

Reply:
Since all other translations are `german => other-lang` I suggest we leave it that way. Like this we could introduce a fourth set of translations in crowdin and everything would be the same.